src/corehost/cli/fx_definition.h

```diff
@@ -24,7 +24,7 @@ class fx_definition_t
     const pal::string_t& get_found_version() const { return m_found_version; }
     const pal::string_t& get_dir() const { return m_dir; }
     const runtime_config_t& get_runtime_config() const { return m_runtime_config; }
-    void parse_runtime_config(const pal::string_t& path, const pal::string_t& dev_path, const runtime_config_t* defaults);
+    void parse_runtime_config(const pal::string_t& path, const pal::string_t& dev_path, const runtime_config_t* previous_config, const runtime_config_t* app_config);
```

Comment:
What does `previous_config` mean? Is there a better name for this?

Reply:
It means the config from the next higher layer which is used to default the current. So nullptr for the app, and the app's config for the first framework.